common/make_migrations.sh

```diff
@@ -27,12 +27,12 @@ docker run \
        --env-file common/environments/dev \
        --interactive \
        --link drdb:postgres\
-       dr_models python3.6 manage.py makemigrations data_refinery_common --noinput
```

Comment:
Why was this removed? This is necessary for running this command in CI. That's why we added it in the first place.